CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [4.2.1](https://pypi.org/project/django-staff-sso-client/4.2.1/) (2023-10-02)
 
```

Comment:
Looks like the change log is in the wrong order. Shouldn't this be above 4.2.1?

Reply:
yes it should - thanks